src/main/java/teammates/ui/request/AccountCreateRequest.java

```diff
@@ -35,6 +37,10 @@ public String getInstructorComments() {
         return this.instructorComments;
     }
 
+    public String getCaptchaResponse() {
+        return this.instructorComments;
```

Comment:
This may have accidentally been left out when this was copy-pasted.

```suggestion
        return this.captchaResponse;
```

Reply:
To try to meet the deadline, I will just try to do this myself. If some check fails, then you may need to be involved. If not, I think this PR should be good to go.

Reply:
Oh yeah, I remember changing it but likely accidentally undid it when trying to undo something else. Thanks for the fix!